doc/requirements.md

```diff
@@ -82,6 +82,16 @@ git clone https://github.com/osism/testbed ~/src/github.com/osism/testbed
 
 :::
 
+## Other needed repositories
+
+The following two repositories are needed as well.
+Keep their copies relative to the testbed path from above:
+
+```sh
+git clone https://github.com/osism/ansible-collection-commons ~/src/github.com/osism/ansible-collection-commons
+git clone https://github.com/osism/ansible-collection-services ~/src/github.com/osism/ansible-collection-services
```

Comment:
There is no need to do this manually, you can run `make prepare` in the top-level of the repo in order to have this performed automatically plus some additional checks.

Reply:
ah, after rereading https://docs.scs.community/docs/testbed/doc/quickstart I had trouble with the docs, obviously.

Imho the problem is, that the language is not really clear there, the docs mix _optional_ commands - which you can run - with _needed_ commands, without explicitly stating that they need to be run.

I just skimmed the quickstart section (in most docs, a quickstart is an optional section in contrast to the "getting started" section), read near the end "you can log in to the manager via SSH." and concluded that all these commands are optional.

I'll provide a different PR to make the language more clear.

also the [getting started](https://docs.scs.community/docs/testbed/doc/getting_started) section does not mention that you need to follow the quickstart section, it only points to the "preparation" section and it's unclear that you then also need to follow up with the quickstart.